specs/Modal.spec.js

```diff
@@ -166,6 +166,7 @@ describe('State', () => {
     expect(
       moverlay(modal).className.includes('myOverlayClass')
     ).toBeTruthy();
+    unmountModal();
```

Comment:
`unmountModal` shouldn't be necessary. `afterEach` should deal with clean up. I'll have a look on this.